### PR TITLE
Add benchmark.js

### PR DESCRIPTION
**Pull request by sigorilla**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,45 +1,78 @@
...
-function test(callback) {
-    var sumTime = 0;
-
-    for (var i = 0; i < ITERATION_COUNT; i++) {
-        var start = process.hrtime();
-        callback();
-        sumTime += process.hrtime(start)[0] * 1000 + process.hrtime(start)[1] / 1000000;
+var tests = {
+    'React + ReactDOMServer': function () {
+        var listView = getListView(React);
```
Comment (alt-j) [inline index.js]: In that way you affect results.
I think, we must do like this https://github.com/alt-j/react-server-benchmark/pull/1/files#diff-168726dbe96b3ce427e7fedce31bb0bcL26

**Comment by alt-j** [comment]
Can you, please, split index into two files: list of tests and runner?

**Comment by sigorilla** [comment]
:up:

File: tests.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        from: React,
+        fn: function (listView, dataSet) {
+            var element = React.createElement(listView, dataSet);
+            return ReactRender.elementToString(element);
+        }
+    },
+
+    'FastReactServer + FastReactRender': {
+        from: ReactServer,
+        fn: function (listView, dataSet) {
```
Comment (alt-j) [inline tests.js]: maybe call it `callback` or `run`?

Comment (alt-j) [reply tests.js]: or `test`?

Comment (sigorilla) [reply tests.js]: Yes, `test` will be good

**Comment by alt-j** [comment]
thank you for contributing 🍻

**Pull request merged**